json-rpc/src/runtime.rs

```diff
@@ -469,11 +491,17 @@ fn set_response_error(
     error: JsonRpcError,
     request_type: &str,
     method: &str,
+    sdk_lang: &SdkLang,
```

Comment:
Change the function signature of this from `&SdkLang` to `SdkLang`. Since SdkLang is Copy you will be able to drop the `&` when passing this type in too.